classes/admin_setting_manage_collectors.php

```diff
@@ -88,26 +88,26 @@ public function is_related($query) {
      */
     public function output_html($data, $query='') {
         global $OUTPUT;
-        $return = '';
 
         $formats = \core_plugin_manager::instance()->get_plugins_of_type('cltr');
 
-        $txt = get_strings(array('settings', 'name', 'enable', 'disable', 'default'));
+        $txt = get_strings(array('settings', 'name', 'enable', 'disable', 'default', 'actions'));
         $txt->uninstall = get_string('uninstallplugin', 'core_admin');
 
         $table = new \html_table();
-        $table->head  = array($txt->name, $txt->enable, $txt->uninstall, $txt->settings);
+        $table->head  = array($txt->name, $txt->uninstall, $txt->actions);
         $table->align = array('left', 'center', 'center', 'center', 'center');
-        $table->attributes['class'] = 'manageformattable generaltable admintable w-auto';
```

Comment:
can you please rebase to keep the w-auto I added

Reply:
done